packages/apps/staking/src/pages/Dashboard/index.tsx

```diff
@@ -7,39 +7,73 @@ import LockedAmountCard from '../../components/LockedAmountCard';
 import PageWrapper from '../../components/PageWrapper';
 import StakedAmountCard from '../../components/StakedAmountCard';
 import WithdrawableAmountCard from '../../components/WithdrawableAmountCard';
-import { OverviewIcon } from '../../icons';
+import { DarkOverviewIcon, OverviewIcon } from '../../icons';
 
 const Dashboard: FC = () => {
-  const theme = useTheme();
-  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+  const { isDarkMode, breakpoints, palette } = useTheme();
+  const isMobile = useMediaQuery(breakpoints.down('sm'));
 
   return (
     <PageWrapper>
-      <Box className="violet-header" flex={1}>
-        <Box mx={{ xs: 4, md: 6, lg: 8 }} display="flex" flexDirection="column">
+      <Box
+        display="flex"
+        flexDirection="column"
+        minHeight="calc(100dvh - 212px)"
+        overflow="hidden"
+        mx={{ xs: -3, sm: 0 }}
+        borderRadius={{ xs: 0, sm: '20px' }}
+      >
+        <Box
+          display="flex"
+          flexDirection="column"
+          height="222px"
+          px={{ xs: 4, md: 6, lg: 8 }}
+          py={{ xs: 2, sm: 4 }}
+          sx={{
+            background: isDarkMode
```

Comment:
Maybe use `bgcolor="string"` here as well?

Reply:
it does not work. i assume it's because `elevation.light` is a gradient